Dockerfile

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is this by accident? Other than that, LGTM

Reply:
Oh I think that happened when I tried to merge master